packages/external-db-config/lib/readers/aws_config_reader.js

```diff
@@ -48,15 +49,18 @@ class AwsConfigReader {
         return { region: this.region, secretKey: process.env.SECRET_KEY, endpoint: process.env.ENDPOINT_URL }
       }
       const cfg = await this.readExternalConfig()
-                            .catch(() => EmptyAWSConfig)
+                            .catch(() => { true})
```

Comment:
what is `{ true }`?

Reply:
typo, I meant `{ error:true }`, it's the empty config of dynamo.